src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -0,0 +1,715 @@
+
+
+def __export_features(retrieved_features, format_):
+    exported_dict = {}
+    client_filters = []
+
+    if format_ in ('yaml', 'properties'):
+        # We only support json feature flags for now
+        logger.warning("Exporting feature flags to a yaml or properties file is not supported yet. Ignoring all feature flags.")
+        return exported_dict
+
+    if format_ == 'json':
+        exported_dict["FeatureManagement"] = {}
+
+    try:
+        # retrieved_features is a list of FeatureFlag objects
+        for feature in retrieved_features:
+
+            # if feature state is on or off, it means there are no filters
+            if feature.state == "on":
+                feature_state = True
+
+            elif feature.state == "off":
+                feature_state = False
+
+            elif feature.state == "conditional":
+                feature_state = {"EnabledFor": []}
+                client_filters = feature.conditions["client_filters"]
+                # client_filters is a list of dictionaries, where all dictionaries have 2 keys - Name and Parameters
+                for filter_ in client_filters:
+                    feature_filter = {}
+                    feature_filter["Name"] = filter_.name
+                    if filter_.parameters:
+                        feature_filter["Parameters"] = filter_.parameters
+                    feature_state["EnabledFor"].append(feature_filter)
+
+            feature_entry = {feature.key: feature_state}
+
+            if format_ == 'json':
+                exported_dict["FeatureManagement"].update(feature_entry)
+
+        return __compact_key_values(exported_dict)
+
+    except Exception as exception:
+        raise CLIError("Failed to export feature flags. " + str(exception))
+
+
+def __convert_feature_dict_to_keyvalue_list(features_dict, format_):
+    # pylint: disable=too-many-nested-blocks
+    key_values = []
+    default_conditions = {'client_filters': []}
+    default_value = FeatureFlagValue(id_="")
+
+    try:
+        if format_ == 'json':
+            for k, v in features_dict.items():
+                key = FEATURE_FLAG_PREFIX + str(k)
+                default_value.id_ = str(k)
+
+                if isinstance(v, dict):
+                    # This is a conditional feature
+                    default_value.enabled = True
+                    try:
+                        default_value.conditions = {'client_filters': v["EnabledFor"]}
+                    except KeyError:
+                        raise CLIError("Feature '{0}' must contain 'EnabledFor' definition or have a true/false value. \n".format(str(k)))
+
+                    if not default_value.conditions["client_filters"]:
+                        # We support alternate format for specifying always OFF features
+                        # "FeatureU": {"EnabledFor": []}
+                        default_value.enabled = False
+                        default_value.conditions = default_conditions
```

Comment:
when every iteration you create a FeatureFlagValue, you can assign the default condition to it. Then in here, you don't need the if check again.